### ISSUE TITLE
I don't know......

### ISSUE THREAD

**Issue by bilibiliwangzheruoyun** [clearlinux/distribution#2913]
HELP ME ... ...
![截图 2023-06-22 12-18-36](https://github.com/clearlinux/distribution/assets/110314418/1db77cff-d152-4fed-9ca7-2f8aba9022f4)

**Comment by bilibiliwangzheruoyun** [comment]
Whywhy?

**Comment by paulcarroty** [comment]
Attach the `installer.log`.

**Comment by bilibiliwangzheruoyun** [comment]
[clr-installer.log](https://github.com/clearlinux/distribution/files/11948517/clr-installer.log)

**Comment by paulcarroty** [comment]
> 2023/07/04 10:32:59 [DBG] Warning: Partition(s) 1 on /dev/sdb have been written, but we have been unable to inform the kernel of the change, probably because it/they are in use.  As a result, the old partition(s) will remain in use.  You should reboot now before making further changes.

**Comment by bilibiliwangzheruoyun** [comment]
![Screenshot from 2023-07-04 11-16-42](https://github.com/clearlinux/distribution/assets/110314418/7c4e401e-bfea-4dda-bf83-34c913294cee)

**Comment by bilibiliwangzheruoyun** [comment]
![Screenshot from 2023-07-04 11-17-41](https://github.com/clearlinux/distribution/assets/110314418/f8a6d491-8e48-480e-a06d-844e4b54e509)

**Comment by bilibiliwangzheruoyun** [comment]
[clr-installer (1).log](https://github.com/clearlinux/distribution/files/11948842/clr-installer.1.log)

**Comment by paulcarroty** [comment]
OK, I see the free space, but something going wrong. Try `LANG=C clr-installer` for English logs.

```
2023/07/04 11:25:46 [WRN] Failed to find EFI firmware, falling back to legacy BIOS for installation.
```
Huhhh, you don't have EFI, so you don't need the EFI partition too. I suggest to test the server image too.

**Comment by bilibiliwangzheruoyun** [comment]
LANG=C clr-installer
Where can use it?↑

**Comment by bilibiliwangzheruoyun** [comment]
Can I use traditional BIOS?

**Comment by paulcarroty** [comment]
> LANG=C clr-installer Where can use it?↑

In terminal.

**Comment by bilibiliwangzheruoyun** [comment]
Oh, I see.

**Comment by bilibiliwangzheruoyun** [comment]
Sometimes the installer can't find my free partition.

**Comment by bilibiliwangzheruoyun** [comment]
![Uploading IMG20230705114715.jpg…]()

**Comment by bilibiliwangzheruoyun** [comment]
![IMG20230705114715](https://github.com/clearlinux/distribution/assets/110314418/da45935b-d89f-4132-8d6d-4d6bf96ba34e)

**Comment by paulcarroty** [comment]
> Can I use traditional BIOS?

Yes: [1](https://community.clearlinux.org/t/does-the-recent-version-of-clear-linux-support-legacy-bios-without-uefi/4540/9), [2](https://clearlinux.github.io/clear-linux-documentation/reference/image-types.html?highlight=bios).

**Comment by bilibiliwangzheruoyun** [comment]
But I clearly have a free partition.
I can find a free partition when you start the computer.

**Comment by paulcarroty** [comment]
> If disk is MBR installation starts and shortly into installation process fails with error updating partition table. ((c)[forum](https://community.clearlinux.org/t/does-the-recent-version-of-clear-linux-support-legacy-bios-without-uefi/4540/9))

**Comment by bilibiliwangzheruoyun** [comment]
It was really fast, and it failed soon.

**Comment by bilibiliwangzheruoyun** [comment]
Then how to change it to GPT?

**Comment by paulcarroty** [comment]
- Grab a flash drive, format to GPT and try to install Clear there to figure out if MBR is the problem. 
- Try a server image: [39540-live-desktop.iso](https://download.clearlinux.org/current/clear-39540-live-desktop.iso), [clear-39540-kvm-legacy.img.xz](https://download.clearlinux.org/current/clear-39540-kvm-legacy.img.xz)
- I'm out ideas.

**Comment by bilibiliwangzheruoyun** [comment]
kvm-legacy.img
Or how to use it?↑

**Comment by paulcarroty** [comment]
https://clearlinux.github.io/clear-linux-documentation/get-started/bootable-usb.html#bootable-usb

**Comment by bilibiliwangzheruoyun** [comment]
Oh, it's really difficult.

**Comment by bilibiliwangzheruoyun** [comment]
https://cdn.download.clearlinux.org/releases/39400/clear/clear-39400-live-server.iso
The link above is broken.I'll try to install it with .img.

**Comment by paulcarroty** [comment]
https://cdn.download.clearlinux.org/current/clear-39540-live-server.iso this one works for me.

**Comment by bilibiliwangzheruoyun** [comment]
Yours is an updated version.
The one on the website should not have been replaced with an updated version yet.

**Comment by bilibiliwangzheruoyun** [comment]
https://cdn.download.clearlinux.org/releases/

**Comment by paulcarroty** [comment]
OK, try them both :)

**Comment by bilibiliwangzheruoyun** [comment]
The one on the official website is broken.

**Comment by bilibiliwangzheruoyun** [comment]
![Screenshot_2023-07-05-12-36-13-47](https://github.com/clearlinux/distribution/assets/110314418/82fabd6a-afb4-46eb-8209-8e971409635b)

**Comment by bilibiliwangzheruoyun** [comment]
Let me try the one you gave me.

**Comment by bilibiliwangzheruoyun** [comment]
Oh,let me try it.
https://cdn.download.clearlinux.org/releases/39540/clear/clear-39540-live-desktop.iso
The website                                                    /39??0/
https://cdn.download.clearlinux.org/releases/39400/clear/clear-39400-live-desktop.iso
